project/looksgreat/README.rst

```diff
@@ -0,0 +1,72 @@
+========================
+SPER: SMT-Based PCB Escape Routing 
+========================
+
+A project that transform PCB escape routing problem to SMT problem and use SMT solver to solve it.
+
+Basic Information
+=================
+
+Github repository: https://github.com/looksgreat/SPER-SMT-Based-PCB-Escape-Routing
+
+The satisfiability modulo theories (SMT) is a more general form of the SAT (satisfiability problem). In addition to boolean variable, it can also deal with integer, and even if-else and many more statement in first order logic.
+
+Problem to Solve
+================
+
+PCB escape routing is one of the problems in EDA. In this problem, we have a multilayer grid map with multiple pins on it, and several fan-out around the border of the grid map. The goal is to find a path for each pin to fan-out, and each path on the same layer should not cross each other. 
+
+The problem is NP-hard, and there are many heuristic algorithms to solve it. However, the heuristic algorithms are not always optimal, and it is hard to find a good heuristic algorithm.
+
+In this project, I will transform the problem to SMT problem, and use SMT solver to solve it. The SMT solver I use here will be Z3(https://github.com/Z3Prover/z3).
+
+Prospective Users
+=================
+
+PCB factory that want to solve PCB routing problem.
+
+The software will contain multiple option, for example, user can decide whether each fan-out's signal is given by user or decide by the software. Moreover, the software offer two modes, one is aim at minimize path length, which will reduce the production cost. The other mode is aim at making each path length as similar as possible, which will reduce the delay difference between each signal.
+
+System Architecture
+===================
+
+This software will be developed in C++ and use the API provided by z3, and use pybind to support python user.
```

Comment:
In your presentation please make it clear what are provided by z3 and what are your contribution.  Without a clear separation I need to assume you will only be making a wrapper, and grade for the coding parts will be limited.